### PR TITLE
solution added

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
- */
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    static final String wLetter = "w";
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:11]: check constant [naming convention](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultList = new ArrayList<>();
+        List<String> lines = new ArrayList<>();
+        try {
+            lines = Files.readAllLines(Paths.get(fileName));
+
+            for (String line: lines) {
+                String[] words = line.toLowerCase().replaceAll("[^a-z]"," ").split(" ");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:20]:
```suggestion
                String[] words = line.toLowerCase().split("\\W+");
```
Try this way. But write an explanation above this comment what \\W+ regex does)

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (margarita-ignatieva) [reply src/main/java/core/basesyntax/FileWork.java]: done

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (margarita-ignatieva) [reply src/main/java/core/basesyntax/FileWork.java]: done

**Pull request closed without merging**